sklearn/cluster/dbscan_.py

```diff
@@ -66,7 +66,9 @@ def dbscan(X, eps=0.5, min_samples=5, metric='minkowski',
         Note that weights are absolute, and default to 1.
 
     random_state: numpy.RandomState, optional
-        The generator used to shuffle the samples. Defaults to numpy.random.
+        The generator used to shuffle the samples, which affects the cluster
+        numbering and cluster assignments of points that are border points to
+        more than one cluster. Defaults to not shuffling. None.
```

Comment:
No, I mean for good or bad, `random_state=None` means use an arbitrary random number generator, while an additional parameter controls whether randomness is used at all! See for instance the `cross_validation` module.